tools/docker/Dockerfile

```diff
@@ -8,21 +8,27 @@ RUN cd ui && npm install && npm run build
 # Build app
 FROM registry.access.redhat.com/ubi8/python-39
 ARG USER_ID=${USER_ID:-1001}
-ARG DEVEL_AE_LIBRARY=0
+ARG DEVEL_AE_LIBRARY=1
 WORKDIR $HOME
 
 USER 0
+RUN yum install -y yum-utils
+RUN yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+RUN yum install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
 RUN pip install ansible && \
     ansible-galaxy collection install git+https://github.com/benthomasson/benthomasson-eda.git#benthomasson/eda/
 COPY . $WORKDIR
 COPY --from=ui-builder /app/ui/dist/ ./ui/dist
-RUN chown -R $USER_ID ./
+#RUN chown -R $USER_ID ./
 
-USER $USER_ID
+#USER $USER_ID
+USER 0
```

Comment:
We should not use root user if it's possible. If the reason is to be able to run docker binary, you can add these steps instead: 
https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

Reply:
I am not able to get it working with a rootless container even with aiodocker.   I'll file an issue for this.

Reply:
I tried on mac with docker and fedora 33 with podman.    Jobs cannot be launched in another container.

Reply:
It only works on fedora 33 and podman with  setenforce 0.

Reply:
It works now with a rootless container on fedora 33 and podman, but requires setenforce 0,  I am not sure why.

Reply:
In fedora, RHEL and systems with SElinux enabled, you have to mount the socket with the `:z` flag if ansible-events is executed from a container. (rootless mode)
For example: 
`podman run --rm -v /run/user/$USER/podman/podman.sock:/var/run/docker.sock:z [...]`